modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -88,46 +91,103 @@ inline std::vector<const char*> getCharNames(const std::vector<std::string>& nam
     return out_vec;
 }
 
-inline void copyToOut(const cv::Mat& in, cv::Mat& out) {
-    GAPI_Assert(in.depth() == CV_32F);
-    GAPI_Assert(in.size == out.size);
-    const float* const inptr = in.ptr<float>();
-    float* const optr = out.ptr<float>();
-    const int size = in.total();
-    for (int i = 0; i < size; ++i) {
+template<typename T>
+void copyToOut(const cv::Mat& in, cv::Mat& out) {
+    const T* const inptr = in.ptr<T>();
+    T* const optr = out.ptr<T>();
+    const size_t size = std::min(out.total(), in.total());
+    for (size_t i = 0; i < size; ++i) {
         optr[i] = inptr[i];
     }
+    if (size < out.total()) {
+        if (optr[size] <= 0) {
+            std::cout << optr[size] << std::endl;
+        }
```

Comment:
Message is unclear, so it can't help with investigation of test failures.

Reply:
Removed.